js/components/frame.js

```diff
@@ -25,16 +25,14 @@ class Frame extends ImmutableComponent {
 
   updateWebview () {
     let src = this.props.frame.get('src')
-    const isAboutURL = isSourceAboutUrl(src)
-    if (isAboutURL) {
-      src = getTargetAboutUrl(src)
-    }
     let contentScripts = ['content/webviewPreload.js']
-    if (this.props.frame.get('location') === 'about:preferences') {
+    // NOTE: use src instead of location here because location is sometimes
+    // not updated when redirecting to about:certerror
+    if (src === 'about:preferences' || src === 'about:certerror') {
```

Comment:
Any idea why `location` would still be the previous URL here? It seems like it should always be the same as `src` when a fresh `about:` page is loaded.